### ISSUE TITLE
LiteLLMModel cannot set api_base

### ISSUE THREAD

**Issue by alexcong** [huggingface/smolagents#20]
```
agent = CodeAgent(
    tools=[search_tool],
    model=LiteLLMModel(
        model_id="ollama/qwen2.5:14b",
        api_base="http://localhost:11434",
    ),
    planning_interval=3,  # This is where you activate planning!
)
```

I got 

`TypeError: LiteLLMModel.__init__() got an unexpected keyword argument 'api_base'`

The doc on [LiteLLMModel](https://huggingface.co/docs/smolagents/v0.1.3/en/reference/agents#smolagents.LiteLLMModel) suggests api_base and api_key are both valid arguments.

**Comment by bilawalriaz** [comment]
Yeah it's broken at the moment - just remove  ` api_base="http://localhost:11434",` and you should be good to go.

**Comment by aymeric-roucher** [comment]
This should be fixed now with release 1.0.0 @alexcong @bilawalriaz!

**Comment by rawwerks** [comment]
just want to clarify, is the structure above correct to override api_base? 

on the docs: 
```
from smolagents import LiteLLMModel

messages = [
  {"role": "user", "content": "Hello, how are you?"},
  {"role": "assistant", "content": "I'm doing great. How can I help you today?"},
  {"role": "user", "content": "No need to help, take it easy."},
]

model = LiteLLMModel("anthropic/claude-3-5-sonnet-latest")
print(model(messages))

class smolagents.LiteLLMModel
[< source >](https://github.com/huggingface/smolagents/blob/v0.1.3/src/smolagents/models.py#L412)

( model_id = 'anthropic/claude-3-5-sonnet-20240620', api_base = None, api_key = None )
```
